cmake/wasi-sdk-sysroot.cmake

```diff
@@ -360,7 +370,7 @@ set(dist_dir ${CMAKE_CURRENT_BINARY_DIR}/dist)
 # Tarball with just `compiler-rt` builtins within it
 wasi_sdk_add_tarball(dist-compiler-rt
   ${dist_dir}/libclang_rt.builtins-wasm32-wasi-${wasi_sdk_version}.tar.gz
-  ${wasi_resource_dir}/lib/wasi)
+  ${wasi_resource_dir}/lib/wasm32-unknown-wasi)
```

Comment:
Is it normall for the target to appear both in the directory name `${wasi_resource_dir}/lib/wasm32-unknown-wasi` *and* in the library name `libclang_rt.builtins-wasm32-wasi`?  It seems somewhat redundant.